apps/frontend/src/pages/dashboard/libs/components/user-history.tsx

```diff
@@ -0,0 +1,41 @@
+import { Loader } from "~/libs/components/components.js";
+import { DataStatus, UserRouteStatus } from "~/libs/enums/enums.js";
+import {
+	useAppDispatch,
+	useAppSelector,
+	useEffect,
+} from "~/libs/hooks/hooks.js";
+import { actions as userRoutesActions } from "~/modules/user-routes/user-routes.js";
+
+import styles from "./styles.module.css";
+import { UserHistoryCard } from "./user-history-card.js";
+
+const UserHistory = (): React.JSX.Element => {
+	const dispatch = useAppDispatch();
+
+	useEffect(() => {
+		void dispatch(userRoutesActions.getAllByUserId(UserRouteStatus.COMPLETED));
```

Comment:
probably getAllByStatus, as you are not passing user id to it
```suggestion
		void dispatch(userRoutesActions.getAllByStatus(UserRouteStatus.COMPLETED));
```